quick_start/index.rst

```diff
@@ -1,12 +1,17 @@
 .. _quick_start_index:
 
 ******************
-Reference
+Quick Start
 ******************
 
+In this section, we're going to see how to build up a simple NStack module from scratch.
+We're going to build a small python module to greet people,
+and then hook it up to a source and a sink.
+
+Make sure you have :doc:`installed NStack </installation>` and let's get going.
+
```

Comment:
Imagine you have no idea what NStack is when you are writing this -- I'd like a description of what we are building outside of the NStack concepts, e.g.

After writing some simple Python which does x, are going to use NStack to productionise our code in the cloud by turning it into a *module* on the NStack platform. Once we have built it, we are going to connect it to input and output data -- what NStack calls *sources* and *sinks* -- so it can xyz.

You get the idea... Idiot-proof it